addons/mail/models/mail_message.py

```diff
@@ -876,7 +876,8 @@ def _message_reaction(self, content, action):
             self._bus_notification_target(),
             "mail.record/insert",
             Store(self, {"reactions": [(group_command, group_values)]})
-            .add(guest or partner, fields=["name", "write_date"])
+            # sudo: mail.guest - access to guest table is allowed
```

Comment:
`guest can broadcast their own name when reacting` would be a more appropriate explanation